internals/secrethub/run.go

```diff
@@ -37,7 +37,7 @@ var (
 	ErrReadEnvFile            = errRun.Code("env_file_read_error").ErrorPref("could not read the environment file %s: %s")
 	ErrEnvDirNotFound         = errRun.Code("env_dir_not_found").Error(fmt.Sprintf("could not find specified environment. Make sure you have executed `%s set`.", ApplicationName))
 	ErrTemplate               = errRun.Code("invalid_template").ErrorPref("could not parse template at line %d: %s")
-	ErrTemplateFile           = errRun.Code("invalid_template_file").ErrorPref("template file '%s' is invalid: %s")
+	ErrParsingTemplate        = errRun.Code("template_parsing_failed").ErrorPref("error while parsing template file '%s': %s")
```

Comment:
We could also make it "error while *processing* template file:", as it seems the error might come from various phases of processing, including, but not limited to parsing.
It could come for example, from looking up the secret mentioned in the file, from reading or parsing the credential to authenticate to the SecretHub API server to do do so etc.

Reply:
Good point!